miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebViewClient.kt

```diff
@@ -14,10 +14,24 @@ internal class MiniAppWebViewClient(
     private val customScheme: String
 ) : WebViewClient() {
 
-    override fun shouldInterceptRequest(
-        view: WebView,
+    override fun shouldInterceptRequest(view: WebView, request: WebResourceRequest): WebResourceResponse? {
+        val interceptedWebRequest = loader.shouldInterceptRequest(request.url)
+
+        interceptMimeType(interceptedWebRequest, request)
+
+        return interceptedWebRequest
+    }
+
+    @VisibleForTesting
+    internal fun interceptMimeType(
+        interceptedWebRequest: WebResourceResponse?,
         request: WebResourceRequest
-    ): WebResourceResponse? = loader.shouldInterceptRequest(request.url)
+    ) {
+        interceptedWebRequest?.let {
+            if (request.url.toString().endsWith("js", true))
+                it.mimeType = "application/javascript"
```

Comment:
Do we need to use any `const val` to put this `"application/javascript"`?

e.g.

```
private companion object {
    const val MIME_TYPE_JS = "application/javascript"
}
```

Reply:
For now, I keep it as in function scope because there is only javascript support. If we have more than one, I will implement it similar to [ios](https://github.com/rakutentech/ios-miniapp/blob/master/MiniApp/Classes/Display/MimeTypes.swift).